package.json

```
@@ -2,12 +2,13 @@
   "name": "mini-typescript",
   "version": "1.0.0",
   "description": "A miniature model of the Typescript compiler",
-  "main": "index.js",
+  "main": "compiler/index.js",
```

I try to use compiler names for things when it's not wordy or confusing. I vote for `built`:
```suggestion
  "main": "built/index.js",
```

---

Cool, agree